synthtool/gcp/partials.py

```diff
@@ -34,13 +37,15 @@ def load_partials(files: List[str] = _DEFAULT_PARTIAL_FILES) -> Dict:
     deprecation_warning: a warning to indicate that the library has been
         deprecated and a pointer to an alternate option
     """
+    if files is None:
+        files = _DEFAULT_PARTIAL_FILES
+    else:
+        files.extend(_DEFAULT_PARTIAL_FILES)
+    res = {}
```

Comment:
res == result?  Mind renaming it result?

Reply:
done